client/signup/steps/plans/index.jsx

```diff
@@ -53,6 +53,18 @@ export class PlansStep extends Component {
 			this.setState( { isDesktop: matchesDesktop } )
 		);
 		this.props.saveSignupStep( { stepName: this.props.stepName } );
+
+		// trigger guides on this step, we don't care about failures or response
+		wp.req.post(
+			'guides/trigger',
```

Comment:
Should we restrict the API call only to the tailored flows? We have helper functions [here](https://github.com/Automattic/wp-calypso/blob/582bb34f8c87badf1daac0cb5fafff3e7a62d047/packages/onboarding/src/utils/flows.ts).

Reply:
Ah nice! Yeah, we should restrict it.